docs/_posts/2024/2024-03-31-scala-io-nantes.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: "post"
+author:
+  - "pauline"
+title: "Duchess at Scala.io"
+date: "2024-04-02"
+categories:
+  - "code"
+  - "event"
+---
+
+| ![Duchess in Scala.io](/assets/sponsors/scalaio.webp){: width="300"} | Duchess was at Scala.io for the 2024 edition in Nantes |
+
+
+A couple of weeks ago was the scala conference in Nantes. This article focus of the scala women speakers present at the event.
+Also Scala.io organizers offered 3 tickets to the Duchess France community.
+
+# Focus on Monica's talk
+
+Great feedback from Monica. She shared with us about her first year of java and gave very good hints.
+
+**Help others to help you**<br/>
+Junior people should focus on providing context, goal, and more generally be great communicators.
+
+**Let us fail**<br/>
+Let junior developers making and owning mistakes.
+Senior people should let junior people (or new comers in the teams) fail so they can take ownership of their mistakes and learn from them. If you're a senior developer don't solve the problem for you're team mates but rather teach the steps to go through to tackle the problem. Enable the team to solve problems, even it takes a little long to solve these problems.
```

Comment:
Correction : Senior people should let junior people (or newcomers in the teams) fail so they can take ownership of their mistakes and learn from them. If you're a senior developer don't solve the problem for your teammates but rather teach them the steps to tackle the problem. Enable the team to solve their own problems, even it takes a little longer.